app.js

```
@@ -245,6 +245,8 @@ class VoiceIsolatePro {
       document.querySelectorAll('.panel').forEach(p => p.classList.toggle('active', p.id === 'tab-' + t.dataset.tab));
     }));
     document.querySelectorAll('.btn-preset').forEach(b => b.addEventListener('click', () => this.applyPreset(b.dataset.preset)));
+    const saveBtn = document.getElementById('saveCustomPresetBtn');
```

ESLint

[error] 'document' is not defined.

(no-undef)